src/Model/LocalDateTime.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace TractorCow\Fluent\Model;
+
+use DateTimeImmutable;
+use DateTimeZone;
+use Exception;
+use IntlDateFormatter;
+use InvalidArgumentException;
+use SilverStripe\ORM\FieldType\DBDatetime;
+
+/**
+ * Stores dates in the same timezone as DBDateTime, but will format them
+ * based on the timezone of another locale.
+ *
+ * Note: getValue() date will NOT be in the specified timezone. Only dates formatted via ->Format()
+ * or ->getLocalValue() will be converted as expected.
+ */
+class LocalDateTime extends DBDatetime
```

Comment:
I'm wondering if it would be possible to keep this available via an extension. Generally, I prefer composition over inheritance as it's a more flexible approach. I'm thinking of a case where `DBDatetime` is subclassed by project's customisation. Is that case using `getLocalTime` effectively removes the customisation.

Reply:
The problem is that there's no way to extend getCustomFormatter. The object has to be completely overwritten in order for a timezone to be injected.

Reply:
I think this is why other frameworks use facades for extensions ;P It'd be easy then. Oh well!

Reply:
Right, all good then.